### PR TITLE
Offline HTML report

### LINKED ISSUE #29: Use in disconnected enviroments
After reviewing a report created by this tool I noticed that HTML pulls in CSS files and fonts from the Internet. There are some use cases where the viewer of the report may not be online. I propose that we vendor the CSS and fonts in the repo so the reports can be used offline.

### PR DESCRIPTION
**Pull request by Honny1**
This PR removes online dependencies in the HTML report. 

Fixes #29

### CODE REVIEW

File: oscap_report/html_report/report_generator.py [hunk 1]
```diff
@@ -1,15 +1,18 @@
+import base64
 import re
 from io import BytesIO
 from pathlib import Path
 
-from jinja2 import Environment, FileSystemLoader
+from jinja2 import Environment, FileSystemLoader, pass_context
```
Comment (jan-cerny) [inline oscap_report/html_report/report_generator.py:6]: [jcerny@thinkpad oscap-report{pr/31}]$ python3 -m oscap_report.cli -o report.html arf.xml 
Traceback (most recent call last):
  File "/usr/lib64/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib64/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/jcerny/work/git/oscap-report/oscap_report/cli.py", line 9, in <module>
    from .html_report.report_generator import ReportGenerator
  File "/home/jcerny/work/git/oscap-report/oscap_report/html_report/report_generator.py", line 6, in <module>
    from jinja2 import Environment, FileSystemLoader, pass_context
ImportError: cannot import name 'pass_context' from 'jinja2' (/usr/lib/python3.9/site-packages/jinja2/__init__.py)

**Review by jan-cerny — COMMENTED** [review]
Do we need to have all the fonts and images there? Can the fonts be replaced by using universal fonts? Are the images used?


What's the size (in kB) of the generated HTML report?

File: oscap_report/html_report/report_generator.py (hunk 1 above)
Comment (Honny1) [reply oscap_report/html_report/report_generator.py:6]: Which version of jinja2 do you use? In minimal requirements is `>= 3.0.1`. This feature name is new in this version. In the older version, this feature is named different. Developers are will drop support old name in the next version. 

I had the same problem which I solved with the update of the `jinja2`. For example, `tox` tests work fine.

Comment (jan-cerny) [reply oscap_report/html_report/report_generator.py:6]: I have python3-jinja2-2.11.3-1.fc33.noarch as I'm still on F33. Now the problem is whether the new version 3.0 will ever be available in RHEL. That's unlikely in RHEL 8. And in RHEL 9, currently RHEL 9 beta ships python-jinja2-2.11.3-4.el9. Due to that, I think you'll have to make that compatible with both jinja2 versions.

**Comment by Honny1** [comment]
@jan-cerny I removed unused files. The worst scenario report which contains over a thousand rules has 20MB now. Before removing unused files report has 24MB.

File: oscap_report/html_report/report_generator.py (hunk 1 above)
Comment (Honny1) [reply oscap_report/html_report/report_generator.py]: Ok, I fixed the compatibility issue.

**Pull request merged**